frontend/src/components/shared/ComparisonDetails.vue

```diff
@@ -16,9 +16,8 @@
           <router-link :to="{ name: 'explorer', params: { model: currentModel.short_name } }">
             <b>{{ currentModel.short_name }}</b>
           </router-link>
-          has
-          {{ comparisonDetails.details['Reaction'].own }} reactions and
-          {{ comparisonDetails.details['CompartmentalizedMetabolite'].own }} metabolites.
+          {{ currentModel.version }} has {{ comparisonDetails.details['Reaction'].own }} reactions
```

Comment:
Really like the addition of version! Maybe we could add that to line 26 and 31 as well, so that the version is specified for all models in this section?

Reply:
Good idea, fixed in 8ad150b.